index.html

```diff
@@ -614,6 +614,35 @@
   </div>
 </div>
 <!-- ends Text watermark -->
+<!-- Burn in timecode-->
+<span data-toggle="modal" data-target=".burn_in_timecode"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Burn in timecode ">Burn in timecode</button></span>
+<div class="modal fade burn_in_timecode" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="well">
+        <h3>Create a burnt in timecode on your image</h3>
+        <p><code>ffmpeg -i <i>input_file</i> -vf drawtext="fontfile=<i>font_path</i>:fontsize=<i>font_size</i>:timecode=<i>starting_timecode</i>:fontcolor=<i>font_colour</i>:box=1 :boxcolor=<i>box_colour</i>:rate=<i>timecode_rate</i>:x=(w-text_w)/2:y=h/1.2" <i>output_file</i></code></p>
```

Comment:
I added a space before `box=1` as the formatting broke in Ubuntu Firefox without it. As the drawtext filter is wrapped in quotes, spacing doesn't matter, but currently my code is inconsistent and I don't know how to fix.
![screenshot from 2015-12-25 01 13 00](https://cloud.githubusercontent.com/assets/1127102/11998740/ea542bbe-aaa4-11e5-867a-d32a9b8a7775.png)